src/EnergyPlus/Coils/CoilCoolingDX.cc

```diff
@@ -171,7 +171,9 @@ void CoilCoolingDX::instantiateFromInputSpec(EnergyPlus::EnergyPlusData &state,
                                                                    DataLoopNode::ObjectIsNotParent);
 
     // Ultimately, this restriction should go away - condenser inlet node could be from anywhere
-    if (!OutAirNodeManager::CheckOutAirNodeNumber(state, this->condInletNodeIndex)) {
+    bool isOK = false;
+    OutAirNodeManager::CheckAndAddAirNodeNumber(state, this->condInletNodeIndex, isOK);
```

Comment:
Not part of this PR, but isOK should be the return value of this function, not a reference parameter.